modules/rhythmoneBidAdapter.js

```diff
@@ -34,10 +31,7 @@ function RhythmoneAdapter (bidManager, global, loader) {
 
   function load(bidParams, url, callback) {
     loader(url, function(responseText, response) {
-      if (response.status === 200)
-      { callback(200, 'success', response.responseText); }
-      else
-      { callback(-1, 'http error ' + response.status, response.responseText); }
+      if (response.status === 200) { callback(200, 'success', response.responseText); } else { callback(-1, 'http error ' + response.status, response.responseText); }
```

Comment:
This is harder to read than before. It doesn't look like `standardjs` implements a `max-len` but I think we should consider it. I guess it doesn't do that because it can't `--fix` that automatically?